### PR TITLE
Feature/tests for examples

### LINKED ISSUE #6: Cover examples with tests

### PR DESCRIPTION
**Pull request by tikon93**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
```
Comment (zmievsa) [inline pyproject.toml:1]: You didn't set a minimal coverage value :(

File: pyproject.toml [hunk 2]
```diff
@@ -83,6 +83,7 @@ omit = [
     "tests/_data/latest/*",
     "tests/_temp/*",
```
Comment (zmievsa) [inline pyproject.toml]: I believe it makes sense to delete the first three lines as they are from Cadwyn.

File: pyproject.toml [hunk 3]
```diff
@@ -83,6 +83,7 @@ omit = [
...
     "tests/test_tutorial/*/schemas/*",
+    "examples/*"
```
Comment (zmievsa) [inline pyproject.toml:83]: Didn't you want to cover examples though?

Comment (tikon93) [reply pyproject.toml:83]: not sure if covering all of them makes sense - it includes a lot of method for reference to how this versioning may be used and beautiful doc :)

**Review by zmievsa — APPROVED** [review]
Looks great. The tests are quite simple so there's not much to comment on. There are minor stylistic issues but I don't see any reason to focus on them now.

File: pyproject.toml (hunk 3 above)
Comment (zmievsa) [reply pyproject.toml:83]: I feel like examples should be thoroughly covered. If something is in examples, then it displays some feature. In order to cover this feature, you cover that part of your example. And if not, then maybe there's a bit too much redundancy in your examples?

**Pull request merged**